ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending_master.ecf

```diff
@@ -40,6 +40,12 @@ set -x
 ############################################################
 # CALL executable job script here
 ############################################################
+# Set up mailing list
+if [[ "${envir}" != "prod" ]]; then
+    MAILTO="nco.spa@noaa.gov"
+fi
```

Comment:
What is this block supposed to achieve.
Does `nco.spa` need to be alerted for a parallel, or a test?